homeassistant/components/honeywell/__init__.py

```diff
@@ -102,18 +103,19 @@ async def _retry(self) -> bool:
         self.devices = devices
         return True
 
-    def _refresh_devices(self):
+    async def _refresh_devices(self):
         """Refresh each enabled device."""
         for device in self.devices:
-            device.refresh()
+            await self._hass.async_add_executor_job(device.refresh)
+            await asyncio.sleep(5)
```

Comment:
Extract the sleep time to a module level constant to make it easy to patch the time to sleep in the tests.

Reply:
That worked great! Thanks for the tip. 👍